graph/types/schema.graphqls

```diff
@@ -12,6 +12,9 @@ type Query {
   # Need to add verification that the caller is the user.
   user(id: ID!): User!
   me: User!
+
+  # Twitter
+  twitterUserHandleAutocompletes(attempt: ID!): [ID!]
```

Comment:
https://delphisproject.slack.com/archives/C01579FP2EP/p1595441510001900 -- Specifically, we want to show, at least, the user's display name and profile photo.